.cirrus.yml

```
@@ -78,6 +78,30 @@ build_task:
       path: "**/test-results/**/*.xml"
       format: junit
 
+validate_external_contributor_build_task:
+  depends_on:
+    - build
+  eks_container:
+    <<: *CONTAINER_DEFINITION_17
+    cpu: 1.5
+    memory: 4G
+  gradle_cache:
+    folder: ~/.gradle/caches
+  maven_cache:
+    folder: ${CIRRUS_WORKING_DIR}/.m2/repository
+  script: |
+    unset ARTIFACTORY_URL && \
+    unset ARTIFACTORY_PRIVATE_USERNAME && \
+    unset ARTIFACTORY_PRIVATE_PASSWORD && \
+    unset ARTIFACTORY_ACCESS_TOKEN && \
+    ./gradlew --refresh-dependencies -x test clean build
```

Let's try with the minimum set of tasks to compile the sources and tests
```suggestion
    ./gradlew assemble testClasses --refresh-dependencies
```